pkg/graphql/spawn.go

```diff
@@ -89,6 +90,9 @@ func NewSpawnerByData(data DtoData, groupName string, instanceTypeName string, i
 	for _, group := range data.User.Groups {
 		spawner.applyVolumeForGroup(groupName, group)
 	}
+	if viper.GetString("jobSubmission.phfsEnabled") == "true" {
```

Comment:
Could we use the same way to pass phfsEnabled from `main.go`

https://github.com/InfuseAI/primehub-controller/blob/master/ee/main.go#L129-L139

It would be better to keep the library class from the configuration logic. Instead, use the constructor to pass the setting.

Reply:
Done. Thx.